### PR TITLE
feat: new tech radar 2024

### PR DESCRIPTION
**Pull request by matlockx**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -51,765 +61,903 @@
...
-          "quadrant": 3,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
-        },
+        },        
         {
-          "label": "parquet",
+          "label": "Swift Package Manger",
```
Comment (binarybro) [inline docs/index.html]: typo in manager, but also we should just use the actual name, as we do with others, eg NPM
```suggestion
          "label": "SwiftPM,
```

or alternatively

```suggestion
          "label": "Swift Package Manager,
```

File: docs/index.html [hunk 2]
```diff
@@ -51,765 +61,903 @@
...
-          "ring": 1,
-          "quadrant": 2,
+          "label": "develocity",
+          "ring": 2,
+          "quadrant": 0,
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://gradle.com/develocity/"
+        },{
+          "label": "angular es build",
```
Comment (binarybro) [inline docs/index.html]:
```suggestion
          "label": "Angular with ESBuild",
```

File: docs/index.html [hunk 3]
```diff
@@ -51,765 +61,903 @@
...
+          "moved": 1,
+          "link": "https://backstage.io/"
+        },
+
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 0, Languages & Tools
+        // HOLD
+        /////////////////////////////////////////////////////
+        {
+          "label": "NPM client",
```
Comment (binarybro) [inline docs/index.html]:
```suggestion
          "label": "NPM",
```

File: docs/index.html [hunk 4]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
-          "quadrant": 2,
+          "quadrant": 1,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "avro",
+          "label": "angular",
```
Comment (binarybro) [inline docs/index.html]: consistency 
```suggestion
          "label": "Angular",
```

File: docs/index.html [hunk 5]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 1,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Service Mesh",
-          "ring": 2,
-          "quadrant": 2,
+          "label": "NX (+ monorepo)",
```
Comment (binarybro) [inline docs/index.html]:
```suggestion
          "label": "Nx",
```

**Review by simonox — APPROVED** [review]

File: docs/index.html [hunk 6]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 1,
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://nextjs.org/"
         },
         {
-          "label": "Webpack Module federation",
-          "ring": 2,
-          "quadrant": 0,
+          "label": "Angular Universal",
```
Comment (binarybro) [inline docs/index.html]:
```suggestion
          "label": "Angular SSR/SSG",
```

**Review by binarybro — APPROVED** [review]
nitpicking only

File: docs/index.html [hunk 7]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 0,
           "active": "TRUE",
-          "moved": -1,
-          "link": ""
+          "moved": 1,
+          "link": "https://codeql.github.com/"
         },
         {
-          "label": "datadog",
+          "label": "Copilot",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "GitHub Copilot",
```

File: docs/index.html [hunk 8]
```diff
@@ -51,765 +61,903 @@
...
+
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 0, Languages & Tools
+        // TRIAL
+        /////////////////////////////////////////////////////    
         {
-          "label": "aws athena",
-          "ring": 0,
-          "quadrant": 3,
+          "label": "appsweep",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "AppSweep",
```

File: docs/index.html [hunk 9]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "github actions",
-          "ring": 1,
-          "quadrant": 2,
+          "label": "develocity",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Develocity",
```

File: docs/index.html [hunk 10]
```diff
@@ -51,765 +61,903 @@
...
+        },{
+          "label": "angular es build",
+          "ring": 2,
+          "quadrant": 0,
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://v17.angular.io/guide/esbuild"
+        },
+        {
+          "label": "backstage",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Backstage",
```

File: docs/index.html [hunk 11]
```diff
@@ -51,765 +61,903 @@
           "link": ""
         },
         {
-          "label": "rust",
-          "ring": 2,
+          "label": "kotlin",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Kotlin",
```

File: docs/index.html [hunk 12]
```diff
@@ -51,765 +61,903 @@
...
+          "ring": 0,
           "quadrant": 0,
           "active": "TRUE",
-          "moved": -1,
+          "moved": 0,
           "link": ""
         },
         {
-          "label": "kotlin",
+          "label": "typescript",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "TypeScript",
```

File: docs/index.html [hunk 13]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
           "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "typescript",
+          "label": "python",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Python",
```

File: docs/index.html [hunk 14]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
           "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "micronaut",
+          "label": "YARN",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Yarn",
```

File: docs/index.html [hunk 15]
```diff
@@ -51,765 +61,903 @@
...
-          "quadrant": 1,
+          "quadrant": 0,
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
         {
-          "label": "angular",
+          "label": "NPM",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "npm",
```

File: docs/index.html [hunk 16]
```diff
@@ -51,765 +61,903 @@
...
-          "quadrant": 1,
+          "quadrant": 0,
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
         {
-          "label": "SSR for small services",
+          "label": "NodeJS",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Node.js",
```

File: docs/index.html (hunk 1 above)
Comment (mattesno1) [reply docs/index.html]:
```suggestion
          "label": "Swift Package Manager",
```

File: docs/index.html [hunk 17]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
-          "quadrant": 3,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "AWS",
+          "label": "gradle",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Gradle",
```

File: docs/index.html [hunk 18]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
-          "quadrant": 2,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "aws s3",
+          "label": "Webpack",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "webpack",
```

File: docs/index.html [hunk 19]
```diff
@@ -51,765 +61,903 @@
...
+          "active": "TRUE",
+          "moved": 1,
+          "link": "https://www.guardsquare.com/appsweep-mobile-application-security-testing"
+        },
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 0, Languages & Tools
+        // ASSESS
+        /////////////////////////////////////////////////////
+        {
+          "label": "rust",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Rust",
```

File: docs/index.html (hunk 3 above)
Comment (mattesno1) [reply docs/index.html]: we have npm in adopt, so this is really about the client only. I'm assuming the CLI?
```suggestion
          "label": "npm CLI",
```

File: docs/index.html [hunk 20]
```diff
@@ -51,765 +61,903 @@
...
+ ////////////////////////////////////////////////////////////
+ 
+
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 1, Frameworks
+        // ADOPT
+        /////////////////////////////////////////////////////
         {
-          "label": "jenkins",
+          "label": "micronaut",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Micronaut",
```

File: docs/index.html [hunk 21]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 1,
           "active": "TRUE",
-          "moved": 1,
-          "link": "https://www.getunleash.io/"
+          "moved": 0,
+          "link": ""
         },
         {
-          "label": "kafka",
+          "label": "Vue for backoffice",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Vue.js for back office",
```

File: docs/index.html [hunk 22]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
-          "quadrant": 0,
+          "quadrant": 2,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Central NPM registry",
+          "label": "datadog",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Datadog",
```

File: docs/index.html [hunk 23]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
           "quadrant": 2,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
+
         {
-          "label": "Core Web Vitals",
+          "label": "unleash",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Unleash",
```

File: docs/index.html [hunk 24]
```diff
@@ -51,765 +61,903 @@
...
-          "quadrant": 3,
+          "quadrant": 2,
           "active": "TRUE",
           "moved": 0,
-          "link": ""
+          "link": "https://www.getunleash.io/"
         },
         {
-          "label": "Accessibility Score",
+          "label": "kubernetes",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Kubernetes",
```

File: docs/index.html [hunk 25]
```diff
@@ -51,765 +61,903 @@
...
-          "quadrant": 3,
+          "quadrant": 2,
           "active": "TRUE",
-          "moved": 1,
+          "moved": 0,
           "link": ""
         },
         {
-          "label": "Firebase",
+          "label": "Github Security",
```
Comment (mattesno1) [inline docs/index.html:492]:
```suggestion
          "label": "GitHub Security",
```

File: docs/index.html [hunk 26]
```diff
@@ -51,765 +61,903 @@
...
           "active": "TRUE",
-          "moved": 1,
+          "moved": 0,
           "link": ""
         },
         {
-          "label": "Lighthouse Score",
-          "ring": 3,
-          "quadrant": 3,
+          "label": "Central NPM registry",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Central npm registry",
```

File: docs/index.html [hunk 27]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 2,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Redis",
-          "ring": 3,
-          "quadrant": 3,
+          "label": "Github",
```
Comment (mattesno1) [inline docs/index.html:540]:
```suggestion
          "label": "GitHub",
```

File: docs/index.html [hunk 28]
```diff
@@ -51,765 +61,903 @@
...
+          "ring": 0,
+          "quadrant": 2,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Angular OAuth, etc",
+          "label": "Github Actions",
```
Comment (mattesno1) [inline docs/index.html:548]:
```suggestion
          "label": "GitHub Actions",
```

File: docs/index.html [hunk 29]
```diff
@@ -51,765 +61,903 @@
...
-          "quadrant": 0,
+          "quadrant": 2,
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
         {
-          "label": "airflow",
+          "label": "graalvm",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "GraalVM",
```

File: docs/index.html [hunk 30]
```diff
@@ -51,765 +61,903 @@
...
-          "quadrant": 3,
+          "quadrant": 2,
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
         {
-          "label": "dbt",
+          "label": "cloudflare",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Cloudflare",
```

File: docs/index.html [hunk 31]
```diff
@@ -51,765 +61,903 @@
...
+        // zoomed_quadrant: 2, Infrastructure
+        // ASSESS
+        /////////////////////////////////////////////////////
+
+
         {
-          "label": "sql server",
-          "ring": 3,
-          "quadrant": 3,
+          "label": "Kafka STRIMZI",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Kafka Strimzi",
```

File: docs/index.html [hunk 32]
```diff
@@ -51,765 +61,903 @@
...
+          "ring": 2,
+          "quadrant": 2,
+          "active": "TRUE",
+          "moved": 0,
+          "link": "https://strimzi.io/"
+        },
+        {
+          "label": "hashicorp vault",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "HashiCorp Vault",
```

File: docs/index.html [hunk 33]
```diff
@@ -51,765 +61,903 @@
...
-          "moved": 0,
-          "link": ""
+          "moved": 1,
+          "link": "https://www.vaultproject.io/"
         },
         {
-          "label": "redshift",
-          "ring": 0,
-          "quadrant": 3,
+          "label": "directus",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Directus",
```

File: docs/index.html [hunk 34]
```diff
@@ -51,765 +61,903 @@
...
           "active": "TRUE",
           "moved": 1,
-          "link": ""
+          "link": "https://directus.io/"
         },
         {
-          "label": "tableau",
-          "ring": 0,
-          "quadrant": 3,
+          "label": "storyblok",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Storyblok",
```

File: docs/index.html [hunk 35]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 2,
           "active": "TRUE",
-          "moved": 0,
-          "link": ""
+          "moved": 1,
+          "link": "https://www.storyblok.com/"
         },
         {
-          "label": "datahub",
+          "label": "argocd",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Argo CD",
```

File: docs/index.html [hunk 36]
```diff
@@ -51,765 +61,903 @@
...
           "active": "TRUE",
-          "moved": 0,
-          "link": ""
+          "moved": 1,
+          "link": "https://argo-cd.readthedocs.io/en/stable/"
         },
         {
-          "label": "Docker",
-          "ring": 0,
+          "label": "HELM charts",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Helm charts",
```

File: docs/index.html [hunk 37]
```diff
@@ -51,765 +61,903 @@
...
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
         {
-          "label": "AWS DMS",
-          "ring": 0,
-          "quadrant": 3,
+          "label": "pulumi",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Pulumi",
```

File: docs/index.html [hunk 38]
```diff
@@ -51,765 +61,903 @@
...
           "active": "TRUE",
           "moved": 1,
-          "link": ""
+          "link": "https://www.pulumi.com/"
         },
         {
-          "label": "MongoDB",
-          "ring": 0,
-          "quadrant": 3,
+          "label": "terraform",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Terraform",
```

File: docs/index.html [hunk 39]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 2,
           "active": "TRUE",
-          "moved": 0,
-          "link": ""
+          "moved": 1,
+          "link": "https://www.terraform.io/"
         },
         {
-          "label": "Container Runtime Interface",
+          "label": "Github Cloud",
```
Comment (mattesno1) [inline docs/index.html:657]:
```suggestion
          "label": "GitHub Cloud",
```

File: docs/index.html [hunk 40]
```diff
@@ -51,765 +61,903 @@
...
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 2, Infrastructure
+        // HOLD
+        /////////////////////////////////////////////////////
+
         {
-          "label": "Spring native",
-          "ring": 1,
-          "quadrant": 1,
+          "label": "jenkins",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Jenkins",
```

File: docs/index.html [hunk 41]
```diff
@@ -51,765 +61,903 @@
...
           "active": "TRUE",
-          "moved": 0,
+          "moved": -1,
           "link": ""
         },
         {
-          "label": "LitElement",
-          "ring": 0,
-          "quadrant": 1,
+          "label": "wordpress/php",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "WordPress/PHP",
```

File: docs/index.html [hunk 42]
```diff
@@ -51,765 +61,903 @@
...
+////////////////////////////////////////////////////////////
+ 
+
+        /////////////////////////////////////////////////////
+        // zoomed_quadrant: 3, Data
+        // ADOPT
+        /////////////////////////////////////////////////////  
         {
-          "label": "Swift",
+          "label": "matomo",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Matomo",
```

File: docs/index.html [hunk 43]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
-          "quadrant": 0,
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Swift UI",
+          "label": "parquet",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Parquet",
```

File: docs/index.html [hunk 44]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 0,
-          "quadrant": 1,
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Swift Package Manger",
+          "label": "aws s3",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Amazon S3",
```

File: docs/index.html [hunk 45]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "CocoaPods",
-          "ring": 3,
-          "quadrant": 0,
+          "label": "aws athena",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Amazon Athena",
```

File: docs/index.html [hunk 46]
```diff
@@ -51,765 +61,903 @@
...
+          "ring": 0,
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Jetpack",
+          "label": "avro",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Avro",
```

File: docs/index.html [hunk 47]
```diff
@@ -51,765 +61,903 @@
...
           "active": "TRUE",
-          "moved": -1,
+          "moved": 0,
           "link": ""
         },
         {
-          "label": "Capacitor",
-          "ring": 2,
-          "quadrant": 1,
+          "label": "kafka",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Kafka",
```

File: docs/index.html [hunk 48]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Xcode Cloud",
-          "ring": 1,
-          "quadrant": 2,
+          "label": "postgres",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "PostgreSQL",
```

File: docs/index.html [hunk 49]
```diff
@@ -51,765 +61,903 @@
...
+          "ring": 0,
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Firebase",
+          "label": "redash",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Redash",
```

File: docs/index.html [hunk 50]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Java",
-          "ring": 3,
-          "quadrant": 0,
+          "label": "airflow",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Airflow",
```

File: docs/index.html [hunk 51]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Cypress",
-          "ring": 3,
-          "quadrant": 1,
+          "label": "redshift",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Amazon Redshift",
```

File: docs/index.html [hunk 52]
```diff
@@ -51,765 +61,903 @@
...
+          "ring": 0,
+          "quadrant": 3,
           "active": "TRUE",
-          "moved": -1,
+          "moved": 0,
           "link": ""
         },
         {
-          "label": "Jest",
+          "label": "tableau",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Tableau",
```

File: docs/index.html [hunk 53]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 1,
-          "link": "https://www.pola.rs/"
+          "link": ""
         },
         {
-          "label": "Consul (Leader Election)",
-          "ring": 3,
+          "label": "google analytics",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Google Analytics",
```

File: docs/index.html [hunk 54]
```diff
@@ -51,765 +61,903 @@
...
           "ring": 2,
-          "quadrant": 2,
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 1,
-          "link": ""
+          "link": "https://aws.amazon.com/de/quicksight"
+        },        
+        {
+          "label": "metabase",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Metabase",
```

File: docs/index.html [hunk 55]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
+          "active": "TRUE",
+          "moved": 0,
+          "link": "https://www.metabase.com/"
         },
         {
-          "label": "ELK for prod",
-          "ring": 3,
-          "quadrant": 2,
+          "label": "superset",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "Superset",
```

File: docs/index.html [hunk 56]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
-          "link": ""
+          "link": "https://superset.apache.org/"
         },
         {
-          "label": "Elastic",
-          "ring": 3,
+          "label": "datahub",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "DataHub",
```

File: docs/index.html [hunk 57]
```diff
@@ -51,765 +61,903 @@
...
+          "quadrant": 3,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "Github",
-          "ring": 0,
-          "quadrant": 2,
+          "label": "sql server",
```
Comment (mattesno1) [inline docs/index.html]:
```suggestion
          "label": "SQL Server",
```

**Review by mattesno1 — COMMENTED** [review]
looked up the spelling of all my suggestions in the respective documentation 🤞

File: docs/index.html (hunk 3 above)
Comment (binarybro) [reply docs/index.html]: Wait NPM should only be in adopt. There is no point in holding it, that's what I mentioned in the session. Didn't we agree on that? And then it should only be NPM, or lowercase, I don't care about that.

**Comment by andimattes** [comment]
@mattesno1 hi man can you reach out on telegram @jsonstef I wanna buy your domain mattes.io

**Pull request merged**